mkdocs.yml

```diff
@@ -19,6 +19,7 @@ markdown_extensions:
 - codehilite
 - admonition
 - pymdownx.tabbed
+    alternate_style: true 
```

Comment:
This doesn't look quite like valid YAML, should it be:
```suggestion
- pymdownx.tabbed:
    alternate_style: true 
```

Reply:
oh man, thanks for that! :facepalm: